README.md

```diff
@@ -17,9 +17,12 @@
 - [Getting Started](#getting-started)
 - [Usage](#usage)
   - [Run](#run)
-  - [List Assets](#list-assets)
-  - [Open, Save, Create, Edit, and Delete Assets](#open-save-create-edit-delete-assets)
+  - [List](#list-command)
```

Comment:
this is cool, however would be nice to see the materials on here that we can also list IE

Assets
Models, XYZ ,, note: the supported materals that we can list right now.

Reply:
@BishoyHanyRaafat lets also double check that these changes are reflected in the help command to ensure consistency